components/org.wso2.carbon.identity.api.user.application/org.wso2.carbon.identity.rest.api.user.application.v1/src/main/resources/META-INF/cxf/application-user-v1-cxf.xml

```diff
@@ -25,6 +25,12 @@
           class="org.wso2.carbon.identity.api.user.application.common.ApplicationServiceHolder">
         <property name="discoverableApplicationManager" ref="identityApplicationServiceFactoryBean"/>
     </bean>
+    <bean id="applicationManagementServiceHolderBean"
+          class="org.wso2.carbon.identity.api.user.application.common.ApplicationServiceHolder">
+        <property name="applicationManagementService" ref="applicationManagementServiceFactoryBean"/>
+    </bean>
     <bean id="identityApplicationServiceFactoryBean"
           class="org.wso2.carbon.identity.api.user.application.common.factory.OSGIServiceFactory"/>
+    <bean id="applicationManagementServiceFactoryBean"
```

Comment:
Make sure the changes are addressed in the relevant REST API component instead of having it in other REST API components.

Reply:
Addressed in [81a8d70](https://github.com/wso2/identity-api-user/pull/129/commits/81a8d7074753b74f8bb43429db8d4ff80383675f)